emulated_hue/discovery.py

```diff
@@ -39,13 +40,16 @@ def start_zeroconf_discovery(config: Config):
     info = ServiceInfo(
         zeroconf_type,
         name=f"Philips Hue - {config.bridge_id[-6:]}.{zeroconf_type}",
-        addresses=[get_ip_pton()],
+        addresses=[config.advertise_ip],
```

Comment:
Looks like it should provide a socket as done with the original code instead of just a plain string.

Reply:
good catch

fixed that